### PR TITLE
Add "PR available" label to the issue marked as solved in the PR description

### LINKED ISSUE #117: Add "PR Available" label when a PR has "Fixes #{issue}"

### PR DESCRIPTION
**Pull request by atomiix**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | When a PR fixes an issue, add the label "PR available" to the issue
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #117
| How to test?      | CI should be green
| Possible impacts? |

### CODE REVIEW

**Review by PierreRambaud — APPROVED** [review]

**Comment by Progi1984** [comment]
@atomiix You can now rebase your PR :)

File: src/AppBundle/PullRequests/Labels.php [hunk 1]
```diff
@@ -27,6 +27,8 @@ final class Labels
 
     const BC_BREAK = 'BC break';
 
+    const PR_AVAILABLE = 'PR available';
```
Comment (Progi1984) [inline src/AppBundle/PullRequests/Labels.php:30]:
```suggestion
    public const PR_AVAILABLE = 'PR available';
```

File: tests/AppBundle/PullRequests/ListenerTest.php [hunk 2]
```diff
@@ -178,6 +198,20 @@ public function getDescriptionTests()
         ];
     }
 
+    public function getPRAvailableTests()
```
Comment (Progi1984) [inline tests/AppBundle/PullRequests/ListenerTest.php:201]:
```suggestion
    public function getPRAvailableTests(): array
```

**Review by Progi1984 — COMMENTED** [review]
Small feedbacks. But it's ok :)

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @atomiix

**Pull request merged**